pkg/core/storage/store.go

```diff
@@ -39,6 +39,8 @@ const (
 	// unfinished state reset and to 0 on unfinished state jump).
 	SYSStateChangeStage KeyPrefix = 0xc4
 	SYSVersion          KeyPrefix = 0xf0
+	// SYSTempStateCheckpoint is used to store the temporary state data during state sync.
+	SYSTempStateCheckpoint KeyPrefix = 0x05
```

Comment:
SYSStateSyncCheckpoint

...is used to store checkpoint of state synchronization process performed by statesync.Module

Something like that. We have a lot of other state-related extensions in the node, I'd like the doc to be precise here because it's easy to forget.